.github/workflows/deploy-docs.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       - name: Deploy docs
         uses: mhausenblas/mkdocs-deploy-gh-pages@master
         env:
+          GEN_REF_PAGES: true
```

Comment:
@frjo @theskumar I believe this is the right place for this? Was between this & putting in one level up under the ubuntu-latest `env`.

Reply:
I believe either would work. Feel right to me.